frontend/www/js/omegaup/components/schools/SchoolOfTheMonth.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="panel panel-default">
+    <div class="panel-heading">
+      <ul class="nav nav-tabs">
+        <li
+          class="active"
```

Comment:
no debería ser algo así como

```suggestion
          v-bind:class="{active: selectedTab === 'candidatesToSchoolOfTheMonth'}"
```

?